File: C/loops/multiplication_table.c
#include <stdio.h>

int main(){
	//Declaração
	int num, result;

	//Entrada
	printf ("Informe um número entre 1 e 10: ");
	fflush (stdout);
	scanf ("%d", &num);

	//Processamento
	while (num < 1 || num > 10) {
		printf ("Informe um número entre 1 e 10: ");
		fflush (stdout);
		scanf ("%d", &num);
	}
	for (int i = 0; i <= 10; i++) {
		result = num * i;
		//Saída
		printf ("\n%d X %d = %d", num, i, result);
	}

}

File: Python/loops/multiplication_table.py
# coding=utf8
# Entrada
num = int (input("Informe um número entre 1 e 10 para ver a tabuada: "))

# Processamento
while num < 1 or num > 10:
    num = int (input("Informe um número entre 1 e 10 para ver a tabuada: "))
print("Tabuada de {0}".format(num))
for i in range (0, 11):
    print("{0} X {1} = {2}".format(num, i, (num * i)))